src/status_im/events.cljs

```diff
@@ -139,13 +139,16 @@
 (handlers/register-handler-fx
  :multiaccounts.login.ui/multiaccount-selected
  (fn [{:keys [db] :as cofx} [_ key-uid]]
-   (let [multiaccount (get-in db [:multiaccounts/multiaccounts key-uid])]
+   ;; We specifically pass a bunch of fields instead of the whole multiaccount
```

Comment:
I would add also in this PR `images` field similarly to the rest, so we would show the image on the login screen which is shown on multiaccounts screen.

Reply:
Added, thank you